packages/insomnia-inso/src/db/index.js

```
@@ -47,9 +47,9 @@ export async function loadDb({
 
   // try load from nedb
   if (!db) {
-    // TODO: Note, unit tests will also try to access the Insomnia Designer app data directory. We should configure this depending on development or production.
     db = await neDbAdapter(
-      appDataDir || envPaths('Insomnia Designer', { suffix: '' }).data,
+      appDataDir ||
+        envPaths(process.env.DEFAULT_APP_DATA_DIR || 'insomnia-app', { suffix: '' }).data,
```

Should we instead throw an error here instead of providing a fallback value for the env? There's no reason it shouldn't exist now, and it seems a bit weird to have a fallback _and_ set the same value in `webpack.config.development.js`.

---

Oh ya, good point. I agree, we should throw an error here. Will update!